src/components/Search.vue

```diff
@@ -0,0 +1,149 @@
+
+<script>
+export default {
+  props: {
+    maxQuickResults: {
+      type: Number,
+      default: 6,
+    },
+    showMore: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  inject: ['girderRest'],
+  data() {
+    return {
+      errText: '',
+      searchText: '',
+      searchMode: 'prefix',
+      searchTypes: ['user', 'folder', 'item'],
+      /* options */
+      searchOptionsMenu: false,
+    };
+  },
+  computed: {
+    quickResults() {
+      return this.searchResults.slice(0, this.maxQuickResults);
+    },
+  },
+  asyncComputed: {
+    searchResults: {
+      default: [],
+      async get() {
+        this.errText = '';
+        try {
+          if (this.searchText) {
+            const { data } = await this.girderRest.get('resource/search', {
+              params: {
+                q: this.searchText,
+                mode: this.searchMode,
+                types: JSON.stringify(this.searchTypes),
+                // add 1 to know if total possible results > maxQuickResults
+                limit: this.maxQuickResults + 1,
+              },
+            });
+            return [].concat(...this.searchTypes.map(t => data[t]));
+          }
+        } catch (err) {
+          this.errText = err.message || 'Unknown error during search.';
+        }
+        return [];
+      },
+    },
+  },
+  methods: {
+    formatUsername(user) {
+      return `${user.firstName} ${user.lastName} (${user.login})`;
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+v-layout.girder-searchbar(row, align-center)
+  v-icon.mdi-24px(color="white") {{ $vuetify.icons.search }}
+  v-menu.grow.mx-3(
+      offset-y,
+      content-class="girder-searchbar",
+      :value="searchText",
+      :nudge-bottom="6")
+    v-text-field(
```

Comment:
@matthewma7 I've found that moving the v-text-field into the activator slot has the undesired effect of bringing up the "no results" popover menu as soon as a user clicks into the text box.

I don't really like showing the user error messages for the initial state.  We could either stick a v-if on the v-list below, or move v-text-field back out of this slot, which would require use of `input-activator`.

I don't know which would be better.

Reply:
Hm, there is an` open-on-click` prop that we can use. 

But overall the reason I meant to use the activator slot is that I think we are trying to use this library as a standard and example for other projects, so I think we could use as much standard stuff as possible. Otherwise, people (like I was) will have questions on why we didn't standard activator because the behavior of full-size input and non-open menu are not explicit and got confused.

Reply:
I certainly agree that we should use standard vuetify features when possible, as long as they make sense to users. 

`open-on-click` works perfectly.  Thank you!